### DISCUSSION TITLE
Using SurrealDB as a database

### DISCUSSION THREAD

**Discussion by vinodh013** [serverpod/serverpod#525] [Ideas]
Hi everyone, Has anybody thought of using surrealdb as a database  
I want to know other people's opinions and thoughts on using a new database like surrealDB

**Comment by yahu1031** [comment]
It's up to you to use it.

**Comment by mozexty** [comment]
SurrealDB No.
I've been thinking about MySQL instead.

________________________________

**Comment by mozexty** [comment]
I took a look at SurrealDB and I can tell it's as ambitious as Serverpod.

They make a great match.

________________________________
